components/Pledge/Merci.js

```diff
@@ -83,6 +84,27 @@ class Merci extends Component {
       ...parseSignInResponseQuery(query)
     }
   }
+
+  componentDidMount () {
+    this.maybeRelocateToOnboarding()
+  }
+
+  componentDidUpdate () {
+    this.maybeRelocateToOnboarding()
+  }
+
+  maybeRelocateToOnboarding () {
+    const { me, query } = this.props
+
+    if (me && ONBOARDING_PACKAGES.includes(query.package)) {
```

Comment:
```suggestion
    if (me && ONBOARDING_PACKAGES.includes(query.package) && !query.claim) {
```
Claim also waits for a me and should win.